python/torch_mlir/dialects/torch/importer/jit_ir/build_tools/upstream_shape_helpers.py

```diff
@@ -216,6 +216,10 @@ def max_pool2d_with_indices(input: List[int], kernel_size: List[int], stride: Li
   out = max_pool2d(input, kernel_size, stride, padding, dilation, ceil_mode)
   return (out, out)
 
+def max_pool2d_with_indices_backward(input: List[int], kernel_size: List[int], stride: List[int], padding: List[int], dilation: List[int], ceil_mode: bool):
```

Comment:
This file should not be edited, except by copying upstream code into it. I just added a note into top fo the file: https://github.com/llvm/torch-mlir/commit/94df096c11011667ddd0da1a5058f7918732b5dc

Reply:
You can just put this code into the main shape_lib_gen.py

Reply:
Sure, I will follow that.